tests/cephadm/test_label_no_schedule.py

```diff
@@ -0,0 +1,73 @@
+from json import loads
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class LabelNoScheduleError(Exception):
+    pass
+
+
+def adding_label(instance, node):
+    """
+    Method add _no_schedule label in the node
+    Args:
+        instance: cluster instance
+        node: cluster node
+    Returns:
+        boolean
+    """
+    log.info(f"Adding '__no_schedule' label in {node}")
+    cmd = f"cephadm shell ceph orch host label add {node} _no_schedule"
+    out, err = instance.installer.exec_command(cmd, sudo=True)
+    if "Added" in out:
+        return 0
+    raise LabelNoScheduleError("Failed to add _no_schedule label to node")
+
+
+def check_daemons(instance, node):
+    """
+    Method checks all the deamons in node
+    Args:
+        node: cluster node
+    Returns:
+        boolean
+    """
+    log.info(f"Checking all the deamons in {node}")
+    cmd = f"cephadm shell ceph orch ps {node} --format json"
+    out, err = instance.installer.exec_command(cmd, sudo=True)
+    all_running_daemons = [data["daemon_type"] for data in loads(out)]
+    if all(item == "osd" for item in all_running_daemons):
+        log.info("All daemons except OSD moved/removed in node2")
+    else:
+        LabelNoScheduleError(
+            "Adding '__no_schedule' label and checking daemons to node2 failed"
+        )
+
+
+def run(ceph_cluster, **kw):
+    """Adding "__no_schedule" label to a node and verify its functionality in a cluster
```

Comment:
Use short description `Test '__no_schedule' lable to node in cluster`

Reply:
Updated